### PR TITLE
applied access checks in home page as well as in KnowledgeCafe/index.blade.php

### PR DESCRIPTION
**Pull request by satendra95**

### CODE REVIEW

**Comment by satendra95** [comment]
@rathorevaibhav 
please review it
#413

(1) Check on home page.
if(auth()->user()->can('weeklydoses.view') || Auth::user()->can('library_books.view'))

(1) Check on KnowledgeCafe page. 
can('weeklydoses.view')
can('library_books.view')

**Comment by satendra95** [comment]
@rathorevaibhav
please review it
#413

(1) Check on home page.
if(auth()->user()->can('weeklydoses.view') || Auth::user()->can('library_books.view'))

(1) Check on KnowledgeCafe page.
can('weeklydoses.view')
can('library_books.view')

File: resources/views/home.blade.php [hunk 1]
```diff
@@ -20,13 +20,13 @@
         </div>
         @endcan
 
-        @can('library_books.view')
+        @if(auth()->user()->can('weeklydoses.view') || Auth::user()->can('library_books.view'))
```
Comment (rathorevaibhav) [inline resources/views/home.blade.php]: use the auth() helper

File: resources/views/knowledgecafe/index.blade.php [hunk 2]
```diff
@@ -4,16 +4,20 @@
 <div class="container">
     <br>
     <div class="d-flex justify-content-start row flex-wrap">
+            @can('library_books.view')
         <div class="col-md-3 card mx-5 my-3">
             <a class="card-body no-transition" href="/knowledgecafe/library/books">
                 <br><h2 class="text-center">Library</h2><br>
             </a>
         </div>
+            @endcan
```
Comment (rathorevaibhav) [inline resources/views/knowledgecafe/index.blade.php]: fix the formatting.

**Comment by satendra95** [comment]
@rathorevaibhav
done with feedback,
please review it
#413

**Comment by satendra95** [comment]
@rathorevaibhav
done with feedback,
please review it
#413

**Comment by rathorevaibhav** [comment]
@satendra95 please make the review changes in `app.blade.php` as well (navbar)

**Pull request merged**